### PR TITLE
[build-deps] prepare release, bump version

### PR DESCRIPTION
**Pull request by micbar**
# Description

- [x] Prepare release
- [ ] Tag
- [ ] Merge back into `edge`

### CODE REVIEW

**Review by kulmann — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -411,6 +513,34 @@ Details
    https://github.com/owncloud/ocis/issues/3073
    https://github.com/cs3org/reva/pull/2977
 
+Changelog for reva 2.7.1 (2022-07-15)
```
Comment (phil-davis) [inline CHANGELOG.md:516]: That's strange - the changelog generator decided to move the 2.7.1 section.

**Review by phil-davis — APPROVED** [review]

File: changelog/NOTE.md [hunk 2]
```diff
@@ -1,82 +1,130 @@
...
+   lock functions. That is wrong, as this function is just a helper to generate the lock file name
+   from a given file to lock.
+
+   https://github.com/cs3org/reva/pull/3204
+
+*   Enhancement #3192: Enable space members to update shares
+
+   Enabled space members to update shares which they have not created themselves.
+
+   https://github.com/cs3org/reva/pull/3192
```
Comment (phil-davis) [inline changelog/NOTE.md:129]: https://app.codacy.com/gh/cs3org/reva/pullRequest?prid=10135807 is too stupid - it complains about "Warn for too many consecutive blank lines", but does not complain about the same layout in other places. This is the only silly thing that causes Codacy to put a "fail" on this PR.

Comment (micbar) [reply changelog/NOTE.md:129]: fixed it

File: CHANGELOG.md (hunk 1 above)
Comment (micbar) [reply CHANGELOG.md:516]: we changed the template

**Pull request merged**